packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -1570,22 +1570,6 @@ const useEchartsCartesianConfig = (
 
                         return `${tooltipHeader}<br/><table>${tooltipRows}</table>`;
                     }
-                    if (
```

Comment:
This code was buggy and redundant, we just need to make sure axislabel is correct (this is what we use in tooltips by default) 

![Screenshot from 2024-01-26 01-12-37](https://github.com/lightdash/lightdash/assets/1983672/420d6c58-e389-4c6e-88d1-5beb11d97e80)